src/index.ts

```diff
@@ -1,25 +1,48 @@
 import { createPinia } from "pinia";
 import { useProductIdentificationStore } from "./store/productIdentification";
+import { useAuthStore } from "./store/auth";
+import Login from "./components/Login";
 import ShopifyImg from "./components/ShopifyImg";
 import { goToOms } from "./utils";
+import piniaPluginPersistedstate from 'pinia-plugin-persistedstate'
 
 // TODO: handle cases when the store from app or pinia store are not available
 // creating a pinia store for the plugin
 const pinia = createPinia();
+pinia.use(piniaPluginPersistedstate)
 
+let getAndSetUserDetails: Function
```

Comment:
Create a separate context variable for each component, otherwise, the current scope will be cluttered with a lot of variables causing confusion with time.

Reply:
Updated